docs/source/methods/track_generation.rst

```diff
@@ -9,7 +9,7 @@ The method of characteristics uses tracks across the geometry to discretize the
 
 .. _cyclic-tracks:
 
-Cylic Tracks
+Cyclic Tracks
 ============
```

Comment:
Does the underlying "=" need to be extended to the full length of this subtitle?